### DISCUSSION TITLE
Is there any way to find the harvester ID using CLI?

### DISCUSSION THREAD

**Discussion by archerallstars** [Chia-Network/chia-blockchain#7800] [Q&A]
The harvester ID can be found in the GUI under `Farm > Show Advanced Options > Your Harvester Network`. 

I can't find the harvester ID by using CLI. Considering that I would need to know my harvester ID with CLI even more than GUI since the harvester can only be run through CLI (you have no option to start only the harvester in the GUI and will be forced to run the full node).

I can use this harvester ID to check whether any of my harvesters are working fine in the pool.

**Comment by azaz111** [comment]
chia rpc farmer get_harvesters_summary
